Corona-Warn-App/src/main/res/values/legal_strings.xml

```diff
@@ -35,4 +35,21 @@
     <string name="submission_no_consent_your_consent_subsection_body_second_part" translatable="false">"<b>The app will share your test result in order to warn other users of official coronavirus apps whom you have encountered. The warning will work in the countries mentioned above. If you provide additional information about the onset of your symptoms, this will also be shared.</b>In the next step, please share your test result (or more precisely: your random IDs from the last 14 days) by tapping on “Share”."</string>
     <!-- YTXT: Body for keys submission no consent text third part-->
     <string name="submission_no_consent_your_consent_subsection_body_third_part" translatable="false">"If you have already warned other users, you can withdraw your consent by deleting the app."</string>
+
+    <!-- XTXT: Body for legal information of the contact diary onboarding screen -->
+    <string name="contact_diary_onboarding_legal_information" translatable="false"/>
+    <!-- XTXT: Title for legal information of the contact diary onboarding screen -->
+    <string name="contact_diary_onboarding_privacy_information_title" translatable="false"/>
```

Comment:
@janetback attached the translations for the legal texts in the ticket. Could please copy them over?

Reply:
Sure thing